packages/block-library/src/image/image.js

```diff
@@ -839,6 +843,30 @@ export default function Image( {
 						</ToolsPanelItem>
 					) }
 					{ dimensionsControl }
+					{ ( aspectRatio || scale ) && url && (
+						<ToolsPanelItem
+							label={ __( 'Focal point' ) }
+							isShownByDefault
+							hasValue={ () => !! focalPoint }
+							onDeselect={ () =>
+								setAttributes( {
+									focalPoint: undefined,
+								} )
+							}
+						>
+							<FocalPointPicker
+								__nextHasNoMarginBottom
+								label={ __( 'Focal point' ) }
+								url={ url }
+								value={ focalPoint }
```

Comment:
A nice detail from Cover block is that it uses `onDrag and onDragStart` events to update the editor preview while dragging. Can we add this here too?